src/Domain.LinnApps/PurchaseOrders/ISupplierKitService.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Linn.Purchasing.Domain.LinnApps.PurchaseOrders
+{
+    using System.Collections.Generic;
+
+    public interface ISupplierKitService
+    {
+        public IEnumerable<SupplierKit> GetSupplierKits(PurchaseOrder order, bool getComponents = false);
```

Comment:
no need for access modifiers in interfaces (I still don't really understand why they even let you put them there)

Reply:
all the other interfaces in that folder have them and if you take them it complains that it wants an access modifier internal or public

Reply:
that can't be right. the public on line 7 should be removable?

Reply:
ah i misunderstood i thought you meant the public on the interface